NVO_conesearch_April2002.html

```diff
@@ -0,0 +1,164 @@
+
```

Comment:
As this isn't used in the build should it (and the v1.03 source) be under a history or archive folder?

Reply:
The NVO document should be there, you're right, I'll move it.
The v.1.03 "source" is actually the same document available on the IVOA official document repository thus probably, at this stage, it can also be removed and a note set in the "history" folder.